README.md

```diff
@@ -9,4 +9,14 @@ I have also changed the code so that it works with the latest version of the [li
 
 I should also add that I am using this project to test different code assistants and see how they work and how to integrate them into a normal workflow.
 
+## Node Descriptions
```

Comment:
Please remove these changes and instead add an entry into the CHANGELOG.md file.